src/main/groovy/com/ullink/ProjectFileParser.groovy

```diff
@@ -40,25 +40,42 @@ class ProjectFileParser {
         }
     }
 
-    def renameExtension(def file, def newExtension) {
-        FilenameUtils.removeExtension(file) + newExtension
+    File getDotnetAssemblyFile() {
+        if (properties.TargetPath) {
+            project.file(properties.TargetPath)
+        }
     }
 
-    File getDotnetAssemblyFile() {
-        project.file(properties.TargetPath)
+    List getDotnetDebugFile() {
+        if (properties.DebugSymbols) {
+            if (eval._DebugSymbolsOutputPath) {
+                eval._DebugSymbolsOutputPath.collect { new File (it.FullPath) }
+            } else {
+                [ new File (outputPath, properties.TargetName + (OperatingSystem.current().windows ? '.pdb' : '.mdb')) ]
+            }
+        }
     }
 
-    File getDotnetDebugFile() {
-        File target = project.file(properties.TargetPath)
-        new File(renameExtension(target.path, OperatingSystem.current().windows ? '.pdb' : '.mdb'))
+    List getDocumentationFile() {
+        if (eval.FinalDocFile) {

```

Comment:
getDotnetDebugFile still have the fallback when _DebugSymbolsOutputPath is not defined no?